app/models/series.rb

```diff
@@ -265,6 +259,14 @@ def word_count
     self.works.posted.pluck(:word_count).compact.sum
   end
 
+  def public_word_count
+    self.works.posted.unrestricted.pluck(:word_count).compact.sum
+  end
+
+  def general_word_count
+    self.works.posted.pluck(:word_count).compact.sum
+  end
```

Comment:
Note: double-check that we're computing this value correctly. I'm not sure where we test visible_word_count, so it's possible we already have some tests? But some edge cases off the top of my head would be series containing:
- draft works
- works with draft chapters
- mystery works (unrevealed)
- anonymous works
- archive-locked works
- works hidden by an admin

(I'm also curious if this value is ever different if the creator of the series is the user looking at the value?)

I call this out for this PR specifically, because if we're about to reindex all series it would be good to make sure we have the right numbers computed, it's comparatively much easier to change the word count display values